### PR TITLE
Add readme & requirements for scripts

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Development scripts for Kubernetes documentation
+
```
Comment (zacharysarah) [inline scripts/README.md]: I'd like to see a section here listing each script with a description of what it does.

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Requirements
+
+Some of those scripts have external requirements you can install them using the following commands:
```
Comment (zacharysarah) [inline scripts/README.md]: Replace with:
> Some of those scripts have external requirements. You can install them with the following commands:

File: scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+python3 -m pip install -r requirements.txt
```
Comment (zacharysarah) [inline scripts/README.md]: Please code-fence this line.

```
    ```
    python3 -m pip install -r requirements.txt
    ```
```

File: scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Options:
+  --reference TEXT  Specify the reference version of the file. Default to the
+                    English one.
+  --git-path TEXT   Specify git path
+  --help            Show this message and exit.
+```
+
+## test_examples.sh
+
+This script will test if your changes will alter examples bundled inside the website.
```
Comment (zacharysarah) [inline scripts/README.md]: Replace with:
> This script tests whether a change affects example files bundled in the website.

File: scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+To install the dependencies.
```
Comment (zacharysarah) [inline scripts/README.md]: nit: s/dependencies./dependencies:

File: scripts/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+    $ ./scripts/test_examples.sh install
+
+To run the examples.
```
Comment (zacharysarah) [inline scripts/README.md]: s/examples./examples:

File: scripts/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+    $ ./scripts/test_examples.sh run
+
+## check-headers-file.sh
+
+This will check the headers if you are in production setup.
```
Comment (zacharysarah) [inline scripts/README.md]: Replace with:
> This script checks the headers if you are in a production environment.

File: scripts/README.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+## hugo-version-check.sh
+
+This script will check whether the local hugo matches the one used on netlify.
```
Comment (zacharysarah) [inline scripts/README.md]: Replace with:
> This script checks whether your local Hugo version matches the version used in production.

**Comment by remyleone** [comment]
@zacharysarah done

**Pull request merged**